packages/httpsnippet-client-api/test/index.test.js

```diff
@@ -1,42 +1,64 @@
 /* eslint-disable global-require */
 /* eslint-disable import/no-dynamic-require */
 /* eslint-disable mocha/no-setup-in-describe */
+const chai = require('chai');
 const { expect } = require('chai');
+const sinon = require('sinon');
+const sinonChai = require('sinon-chai');
+const vm = require('vm');
+const rimraf = require('rimraf');
 const fs = require('fs/promises');
 const HTTPSnippet = require('@readme/httpsnippet');
 const path = require('path');
+const nock = require('nock');
 const client = require('../src');
 const readme = require('@readme/oas-examples/3.0/json/readme.json');
 
+chai.use(sinonChai);
+
 const DATASETS_DIR = path.join(__dirname, '__datasets__');
 const SNIPPETS = require('fs').readdirSync(DATASETS_DIR);
 
 async function getSnippetDataset(snippet) {
+  const SNIPPET_DIR = path.join(DATASETS_DIR, snippet);
+
   const har = await fs
-    .stat(path.join(DATASETS_DIR, snippet, 'har.json'))
-    .then(() => require(path.join(DATASETS_DIR, snippet, 'har.json')))
+    .stat(path.join(SNIPPET_DIR, 'har.json'))
```

Comment:
There's a lot of boilerplate in this method, if you have ideas for how to clean it up I'm all 👂

Reply:
assigning `path.join(SNIPPET_DIR, 'har.json')` to a variable and reusing that a bit might help